sandy/fy.py

```diff
@@ -658,13 +645,17 @@ def _gls_setup(decay_data, kind, index_columns):
 
     """
     if kind == 'cumulative':
-        S = decay_data.get_qmatrix()
+        S = decay_data.get_qmatrix().T
     elif kind == 'chain':
-        S = decay_data.get_decay_chains()
+        S = decay_data.get_transition_matrix()
```

Comment:
It is not `get_transition_matrix`. We have to develop it! Don't implement it for the moment!